File: app/controllers/comments_controller.rb
class CommentsController < ApplicationController
  def index
    @comments = Comment.all
    render json: @comments
  end

  def create
    @comment = Comment.create(name: params[:comment][:name], email: params[:comment][:email], content: params[:comment][:content], station_id: params[:station_id])
    if @comment.save
      redirect_to comments_path
    end
  end

  def new
  end

end

File: app/assets/javascripts/controllers/comments_controller.js
stationApp.controller('CommentsController', ['$scope', '$stateParams', 'CommentFactory', function($scope, $stateParams, CommentFactory) {
  var ctrl = this;
  ctrl.id = $stateParams.id

  this.createComment = function() {
    ctrl.comment = {
      name: this.name,
      email: this.email,
      content: this.content
    }
    return CommentFactory.createComment(ctrl.comment, ctrl.id)
  }
}])
